tests/test_send_plot.py

```diff
@@ -1,4 +1,6 @@
 from agentMET4FOF.agents import AgentMET4FOF, AgentNetwork, MonitorAgent
+import matplotlib
+matplotlib.use('Agg')
```

Comment:
Just for me: what does this do?

Reply:
This sets the backend of matplotlib to Agg, which is well suited for saving plot files, without opening any plotting windows explicitly. 

Without this, there will be eventually errors as we access the default matplotlib GUI backend via multiple threads (agents). Since as soon as we call pyplot.plot, the agents open up the windows.